starknet_py/constants.py

```diff
@@ -1,11 +1,29 @@
-# Address came from starkware-libs/starknet-addresses repository: https://github.com/starkware-libs/starknet-addresses
-FEE_CONTRACT_ADDRESS = (
-    "0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7"
-)
+# pylint: disable=line-too-long
+# fmt: off
+
+from starknet_py.common import int_from_bytes
 
-DEFAULT_DEPLOYER_ADDRESS = (
-    "0x041a78e741e5aF2fEc34B695679bC6891742439f7AFB8484Ecd7766661aD02BF"
-)
+# Address came from starkware-libs/starknet-addresses repository: https://github.com/starkware-libs/starknet-addresses
+FEE_CONTRACT_ADDRESS = "0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7"
+DEFAULT_DEPLOYER_ADDRESS = "0x041a78e741e5aF2fEc34B695679bC6891742439f7AFB8484Ecd7766661aD02BF"
 
 RPC_INVALID_MESSAGE_SELECTOR_ERROR = 21
 RPC_CLASS_HASH_NOT_FOUND_ERROR = 28
+
+DEFAULT_ENTRY_POINT_NAME = "__default__"
+DEFAULT_L1_ENTRY_POINT_NAME = "__l1_default__"
+DEFAULT_ENTRY_POINT_SELECTOR = 0
+DEFAULT_DECLARE_SENDER_ADDRESS = 1
+MASK_250 = 2**250 - 1
```

Comment:
Is it used in more than 1 place? I'm not sure if it makes sense to define the mask as package constant.